control-plane/slimctl/internal/controller/validator.go

```diff
@@ -0,0 +1,36 @@
+package controller
+
+import (
+	_ "embed"
+	"log"
+
+	"github.com/xeipuuv/gojsonschema"
+)
+
+//go:embed client-config.schema.json
```

Comment:
Maybe here we can add a `go:generate`  statement that would either download the schema or get the schema from the config crate, WDYT?

Reply:
As of now will add a cp command in the go:generate